realm/build.gradle

```diff
@@ -14,6 +15,7 @@ buildscript {
         classpath 'com.github.rholder:gradle-one-jar:1.0.4'
         classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.2'
         classpath 'com.netflix.nebula:gradle-extra-configurations-plugin:3.0.3'
+        classpath "io.realm:realm-transformer:${file('../version.txt').text.trim()}"
```

Comment:
I believe you can do this 
```grovvy
buildscript {
    ext.realm_version = file('../version.txt').text.trim()
    repositories {
    }
    dependencies {
        classpath "io.realm:realm-transformer:$realm_version"
    }
}
```
inspired from https://medium.com/@CodingDoug/kotlin-android-a-brass-tacks-experiment-part-1-3e5028491bcc#.f56c6hkt6